modules/nw-ovn-kubernetes-migration-about-live.adoc

```diff
@@ -0,0 +1,166 @@
+= OpenShift SDN live migration overview
+
+The OpenShift SDN live migration is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin. The live migration method offers the following benefits: 
+
+* Continuous service availability
+* Minimized downtime
+* Automatic node rebooting
+* Seamless transition from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin. 
+
+The live migration method is valuable for deployment types that require constant service availability, such as Azure Red Hat OpenShift (ARO), Red Hat OpenShift Dedicated, or Red Hat OpenShift Service on AWS. It is not available for HyperShift deployment types. 
```

Comment:
@stevsmit The migration is not supported in ARO yet in 4.15 due to https://issues.redhat.com/browse/ARO-4352

Reply:
Fixed this one.